app/src/main/java/com/greenfox/fuchsit/zerdareader/server/MockServer.java

```diff
@@ -14,6 +17,11 @@
 
 public class MockServer implements ReaderApiInterface {
     @Override
+    public Call<ArrayList<NewsItem>> getNewsItems() {
+        return null;
```

Comment:
@regnisalram create a mock version for this API call as well, and return a dummy list just as a dummy user is returned below